command/oauth/cmd.go

```diff
@@ -404,11 +415,26 @@ func disco(provider string) (map[string]interface{}, error) {
 	return details, err
 }
 
+func (o *oauth) NewServer() (*httptest.Server, error) {
+	l, err := net.Listen("tcp", o.CallbackListener)
+	if err != nil {
+		return nil, err
```

Comment:
Can you wrap using `errors.Wrap(f)` errors coming from the standard packages. This help us to get traces from errors.